core/arch/arm/tee/pta_socket.c

```diff
@@ -0,0 +1,330 @@
+/*
```

Comment:
Should this file be under `core/arch/arm/sta`? (I don't think we have any pseudo-TA outside of this location).